iree/hal/vulkan/BUILD

```diff
@@ -181,7 +176,7 @@ cc_test(
     deps = [
         ":status_util",
         ":dynamic_symbols",
-        "//iree/base:status_matchers",
+        "//iree/testing:gtest",
```

Comment:
This is already included in PLATFORM_VULKAN_TEST_DEPS, it should not be included twice:

https://github.com/google/iree/blob/6132bb38e63faefc6a244b938174db2f92fd1291/iree/build_defs.oss.bzl#L50-L53